node_server/app.ts

```diff
@@ -39,12 +40,36 @@ function setupLogger(): void {
 }
 
 setupLogger();
-app.use(cors());
+app.use(
+  cors({
+    origin: (origin, callback) => {
+      callback(null, true);
+    },
+    preflightContinue: true,
+  })
+);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
+// eslint-disable-next-line prefer-const
```

Comment:
This can be a const. In JavaScript when an object is set to const, only the root level of the object can't be reassigned. Sub-properties can still be reassigned.

Reply:
Done https://github.com/PointSpire/PointSpire/pull/23/commits/93e14c65624f3d1c9e27c222f0c011a64666c336